lib/display.ex

```diff
@@ -21,9 +21,24 @@ defmodule Display do
   defp name(module), do: name(Atom.to_string(module))
 
   def show_failure(failure, module, name) do
+    intro(module, Tracker.visited)
```

Comment:
Why is the intro printed on failure? I'm sure it works as you expect, but something about that seems weird.